.travis.yml

```diff
@@ -2,9 +2,15 @@ language: php
 php:
   - "5.4"
   - "5.5"
+  - "5.6"
+  - "nightly"
```

Comment:
I actually would use 7.0. It is the same thing, but when the 7.0 is released we don't need to update the travis file. Also, 7.0 is already in beta, it is stable enough to keep as a required environment to test IMO.